### DISCUSSION TITLE
Using Sqlalchemy for connecting Postgresql in a django app

### DISCUSSION THREAD

**Discussion by Rhythmbat205** [sqlalchemy/sqlalchemy#8735] [Usage Questions]
I connected to a PostgresQl db(AWS RDS) using SQLAlchemy in a django app view. Now I want to perform some queries on the tables of the db. When I click on one table for executing the queries, the app doesn't take that table rather it takes the latest table created with the latest schema. Means everytime if i create a new schema with a new table, the app is taking that latest table only even if I am clicking on the old tables. How to resolve this...

**Comment by CaselIT** [comment]
Hi,

I don't know what clicking means in this context.
Regarding the schema, you can specify the schema of a table in the Table itself or on the MetaData
